frontend/src/features/permits/pages/Application/components/pay/CVPayInPersonInfo.tsx

```diff
@@ -0,0 +1,33 @@
+import "./CVPayInPersonInfo.scss";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faExternalLink } from "@fortawesome/free-solid-svg-icons";
+
+export const CVPayInPersonInfo = () => {
+  return (
+    <div className="cv-pay-in-person-info">
+      <h3 className="cv-pay-in-person-info__heading">
+        Pay In-person (Cash/Cheque/Debit)
+      </h3>
+      <p className="cv-pay-in-person-info__body">
+        Pay in person at the <strong>Provincial Permit Centre</strong> <br></br>
+        Toll-free: <strong>1-800-559-9688</strong> or Email:{" "}
+        <strong>ppcpermit@gov.bc.ca</strong>
+        <br />
+        <br />
+        Pay in person at a <strong>Service BC Office (GA)</strong>
+      </p>
+      <a
+        href="https://www2.gov.bc.ca/gov/content/governments/organizational-structure/ministries-organizations/ministries/citizens-services/servicebc#locations"
```

Comment:
You can move this link url into frontend/src/routes/constants.ts, possibly into ONROUTE_WEBPAGE_LINKS

Reply:
and reuse it here